### DISCUSSION TITLE
Amazon Linux 2022 / Corretto and libjpeg.so shared library

### DISCUSSION THREAD

**Discussion by cst-cfalling** [amazonlinux/amazon-linux-2022#173] [General]
Hello, this is my first time interacting with Amazon via github so I apologize if I didn't follow the correct process..

I created a new Issue ( #171 ) that describes the behavior that I am seeing in the current version of Amazon Linux 2022 that did not exist in prior preview releases.  In particular this relates to how images are being rendered in a particular Java library (PDFBox) which is dependent on the Java modules for image processing.  In the earlier preview release, OpenJDK included a library as part of the JVM (libjavajpeg)  This library references the base shared library libjpeg.so.62.   This looks like it used be provided by the package turbojpeg, but it is no longer included when installing that package, as such calls are now failing with an UnsatisfiedLinkError 

java.lang.UnsatisfiedLinkError: /usr/lib/jvm/java-11-amazon-corretto.x86_64/lib/libjavajpeg.so: libjpeg.so.62: cannot open shared object file: No such file or directory

I am working with my team to get a small test case program created if that will help

**Comment by cst-cfalling** [comment]
[PdfChecker-test.zip](https://github.com/amazonlinux/amazon-linux-2022/files/9429329/PdfChecker-test.zip)
Example Test code  
Gradle build project using PDFBox.  Will read a PDF file as an argument and resize the first page. exhibiting the bug as described in current release candidate of Amazon Linux 2022, but not the previous version

**Reply by cst-cfalling** [reply]
FYI attached same zip file to the JIRA issue

**Comment by stewartsmith** [comment]
Just following up on this one, there's a workaround, and a fix is in flight.

Thanks for the report and reproduction case!
